src/extractLoader.js

```diff
@@ -28,9 +28,7 @@ const rndPlaceholder = "__EXTRACT_LOADER_PLACEHOLDER__" + rndNumber() + rndNumbe
 function extractLoader(content) {
     const callback = this.async();
     const options = getOptions(this) || {};
-    const publicPath = options.publicPath ||
-        (this.options && this.options.output && this.options.output.publicPath) ||
-        (this._compilation && this._compilation.outputOptions && this._compilation.outputOptions.publicPath);
+    const publicPath = options.publicPath || getPublicPath(this);
```

Comment:
If `options.publicPath` was an empty string, the empty string should be used. Maybe change that line to:

```js
const publicPath = "publicPath" in options ? options.publicPath : getPublicPath(this)
```

Or maybe put that logic into `getPublicPath` as well?

Reply:
The idea behind `getPublicPath()` was a function that can be removed when we work on #35 .
That's why options was not included.